pkg/sql2pgroll/alter_table.go

```diff
@@ -155,6 +158,32 @@ func convertAlterTableAddUniqueConstraint(stmt *pgq.AlterTableStmt, constraint *
 	}, nil
 }
 
+// convertAlterTableDropColumn converts SQL statements like:
+//
+// `ALTER TABLE foo DROP COLUMN bar
+//
+// to an OpDropColumn operation.
+func convertAlterTableDropColumn(stmt *pgq.AlterTableStmt, cmd *pgq.AlterTableCmd) (migrations.Operation, error) {
+	switch cmd.Behavior {
+	case pgq.DropBehavior_DROP_RESTRICT, pgq.DropBehavior_DROP_BEHAVIOR_UNDEFINED:
+		// Supported
+	case pgq.DropBehavior_DROP_CASCADE:
+		// Fall back to SQL
+		return nil, nil
+	}
+
+	// IF EXISTS not supported
+	if cmd.MissingOk {
+		return nil, nil
+	}
```

Comment:
For other operations we have the pattern of pulling out these checks into `canConvert` functions:

https://github.com/xataio/pgroll/blob/main/pkg/sql2pgroll/alter_table.go?plain=1#L161-L188

It's up to you if you want to follow the pattern here.

Reply:
Done